### ISSUE TITLE
Please put in cfg on your mod desktop separation

### ISSUE THREAD

**Issue by MairanMC** [codetaylor/artisan-worktables#169]
Please put in cfg on your mod desktop separation
Need to split workbenches for my modpack

**Comment by codetaylor** [comment]
Can you please elaborate?

I don't understand what you mean by desktop separation.

What do you mean by split workbenches?

**Comment by codetaylor** [comment]
https://youtu.be/z5X4PTohjks

**Comment by VariableVixen** [comment]
Oh! The tabs on top of the interface that allow access to adjacent tables. They'd like a config option to remove that, because in their modpack they can restrict access to tables (based on skills, from the video) but the tabs bypass that completely.

**Comment by codetaylor** [comment]
This has been implemented and will be in `1.22.0`.

Here is an example config and a video of the result.

```
    # Set to false to disable the joined tabs for workshops.
    B:ENABLE_TABS_WORKSHOPS=true

    # Set to false to disable the joined tabs for workstations.
    B:ENABLE_TABS_WORKSTATIONS=true

    # Set to false to disable the joined tabs for worktables.
    B:ENABLE_TABS_WORKTABLES=false
```

![2019-07-25_12-50-49](https://user-images.githubusercontent.com/1240716/61904053-e33c6380-aeda-11e9-9dac-34c376a7b3eb.gif)
